src/applications/static-pages/wizard/tests/bdd-526.unit.spec.jsx

```diff
@@ -130,8 +130,10 @@ describe('Form 526 Wizard', () => {
     });
 
     expect(wrapper.find('.input-error-date')).to.exist;
+    // This is a key for an i18next translation value - real text will be used
+    // in a non-testing environment
     expect(wrapper.find('.usa-input-error-message').text()).to.contain(
-      'year between',
+      'year-range',
```

Comment:
I think somewhere in the testing environment `i18next` ends up using the [`cimode` language](https://www.i18next.com/overview/api#changelanguage). Here is the [value that's being used](https://github.com/department-of-veterans-affairs/component-library/blob/41286ba6f7e9bde1281006191ac0a042fbf54d67/packages/core/src/i18n/translations/en.js#L9).